src/index.js

```diff
@@ -0,0 +1,18 @@
+import form from './create-form';
+import { projectMethods, projectsArray } from './project';
+import { todoListMethods, toDoArray } from './to-do';
+const date = require('date-input-polyfill');
+import './styles.css';
+
+form.popUp();
+projectMethods.projectSubmission();
+console.log(projectsArray);
```

Comment:
No `console.log` committed to the repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png) from Pathwright)